.github/workflows/cicd.yml

```diff
@@ -139,9 +143,15 @@ jobs:
 
       - name: 🚀 Deploy to Cloud Run
         id: deploy
-        uses: google-github-actions/deploy-cloudrun@v0.7.0
+        uses: google-github-actions/deploy-cloudrun@v0.10.1
         with:
           service: ${{ env.SERVICE_NAME }}
           image: ${{ env.IMAGE }}:latest
           region: ${{ env.REGION }}
           env_vars: WEB_API_KEY=${{ secrets.WEB_API_KEY }}
+          flags: |
```

Comment:
**suggestion**: I suggest adding a request timeout and lowering the memory
<img width="429" alt="image" src="https://user-images.githubusercontent.com/325813/187807485-e4bd9c2a-eb41-483b-9446-81e4701bc032.png">

Reply:
Oh yes, I forgot the timeout. I added a 60s timeout and lowered the memory to 128.